sys/stdio_udp/stdio_udp.c

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (C) 2023 ML!PA Consulting GmbH
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     sys_stdio_udp
+ * @{
+ *
+ * @file
+ * @brief       STDIO over UDP implementation
+ *
+ * @author      Benjamin Valentin <benjamin.valentin@ml-pa.com>
+ *
+ * @}
+ */
+
+#include <errno.h>
+
+#include "net/sock/udp.h"
```

Comment:
```suggestion
#include "macros/utils.h"
#include "net/sock/udp.h"
```

This provides the `MIN()` macro :)